src/main/scala/com/databricks/spark/redshift/FilterPushdown.scala

```diff
@@ -46,6 +46,7 @@ private[redshift] object FilterPushdown {
      getTypeForAttribute(schema, attr).map { dataType =>
        val sqlEscapedValue: String = dataType match {
          case StringType => s"\\'${value.toString.replace("'", "\\'\\'")}\\'"
+         case DateType | TimestampType => s"\\'$value\\'"
```

Comment:
In 1.5+, we'll return java.sql.Date and java.sql.Timestamp, which have locale-independent toString methods, so I think I just have to change this to handle the 1.4.x case and then I think we should be good.